Hindawi Publishers.js

```diff
@@ -49,6 +49,11 @@ function detectWeb(doc,url) {
 	return false;
 }
 
+function decodeEntities(doc, encodedString) {
```

Comment:
@aurimasv -- I know you're super busy, but if you could take 2mins to look at this? Seems like this should be save because we're in a sandbox, but still want to be sure. Couldn't find a method of converting the html entitites without this, but maybe overlooking something simple.